### PR TITLE
use latest version of arize

### PR DESCRIPTION
**Pull request by logan-markewich**
Update the arize callback handler to use newer instrumentation

### CODE REVIEW

File: llama-index-integrations/callbacks/llama-index-callbacks-arize-phoenix/llama_index/callbacks/arize_phoenix/base.py [hunk 1]
```diff
@@ -4,6 +4,29 @@
...
+        from opentelemetry.sdk.trace.export import SimpleSpanProcessor
+
+        endpoint = kwargs.get("endpoint", "http://127.0.0.1:6006/v1/traces")
+        tracer_provider = trace_sdk.TracerProvider()
+        tracer_provider.add_span_processor(
+            SimpleSpanProcessor(OTLPSpanExporter(endpoint))
+        )
+
+        LlamaIndexInstrumentor().instrument(
+            tracer_provider=kwargs.get("trace_provider", tracer_provider)
```
Comment (mikeldking) [inline llama-index-integrations/callbacks/llama-index-callbacks-arize-phoenix/llama_index/callbacks/arize_phoenix/base.py:23]:
```suggestion
            tracer_provider=kwargs.get("tracer_provider", tracer_provider)
```

Comment (axiomofjoy) [inline llama-index-integrations/callbacks/llama-index-callbacks-arize-phoenix/llama_index/callbacks/arize_phoenix/base.py:23]:
```suggestion
            tracer_provider=kwargs.get("tracer_provider", tracer_provider)
```

Comment (logan-markewich) [reply llama-index-integrations/callbacks/llama-index-callbacks-arize-phoenix/llama_index/callbacks/arize_phoenix/base.py:23]: Good catch!

File: llama-index-integrations/callbacks/llama-index-callbacks-arize-phoenix/llama_index/callbacks/arize_phoenix/base.py [hunk 2]
```diff
@@ -4,6 +4,29 @@
...
+        )
+    except ImportError:
+        # using an older version of arize
+        pass
+
+    # older versions of arize, v1.x
```
[the remark this answers is not in the document]
Comment (logan-markewich) [reply llama-index-integrations/callbacks/llama-index-callbacks-arize-phoenix/llama_index/callbacks/arize_phoenix/base.py:29]: yes!

**Pull request merged**